src/components/CalendarDay.jsx

```diff
@@ -23,7 +23,7 @@ const propTypes = forbidExtraProps({
   onDayClick: PropTypes.func,
   onDayMouseEnter: PropTypes.func,
   onDayMouseLeave: PropTypes.func,
-  renderDay: PropTypes.func,
+  renderDayContents: PropTypes.func,
```

Comment:
hm - i'm not sure this prop actually has to change. Is the name that much clearer that it's worth the breaking change now?

Reply:
This change is already breaking because I've taken the opportunity to restructure the `CalendarDay` component. I think with the presence of a `renderCalendarDay` prop, `renderDay` is confusing and we should take this opportunity to clarify what that prop actually does.

Reply:
master might already be breaking, but *this PR* doesn't have to be, and wouldn't be without this prop being renamed.

The naming confusion is a fair point, but `CalendarDay` doesn't take a `renderCalendarDay` prop - the prop names don't need to be the same across all our components.

Reply:
Sorry I think in my late night Friday shuffling around, I lost a change in this component that will most definitely making this PR breaking. I don't think master is breaking right now. 

I maintain that `renderDayContents` is a much more meaningful prop name (in this component and in others) and is not going to be the only breaking change in this PR.